README.md

```diff
@@ -1,77 +1,373 @@
+
+Whether you're looking for a new job or want to meet other developers, Dev.to is an essential part of any aspiring developer's toolkit!
+ 
+At #VetsWhoCode We Believe that long-form writing is one of the best ways to scale yourself, so you don't have to retain information, showcase how you communicate, and a simple way to advocate for yourself to non-technical employers when you're not writing since everyone on here is in tech so the people will be your audience.
+
+### Command Line
+
+The command line is a text-based interface for interacting with your computer. It's where you can type in commands to perform tasks like copying files and managing folders and running applications.
+ 
+It's often called the "command prompt" or "terminal," but it's also known as the "console" or "CLI," which stands for "command line interface." The application name used to access the command line differs from OS to OS: On Windows, it's called cmd (for Command Prompt), while on Mac, it's called Terminal.
+
+Following will be the directions to operate in the command line for MacOS and Windows.
+
+### Command Line Operations 
+
+#### Open
+You can use the Terminal app to access your computer's command line. The Terminal is one of the most powerful tools on your Mac, and it's how we'll be running our code.
+
+MacOS: To open Terminal, go to Applications, then to Utilities finally to Terminal.
+Windows: To open the command prompt, click Start and then click All Programs. In the programs list, click Accessories. In the accessories list, click Command Prompt.
+
+#### Print Working Directory 
+
+To help you know where you are currently in the Terminal we need to print the wordking directory. A directory would look like a folder on the screen of the GUI( Graphical User Interface). Type this command and hit enter:
 
-Below is a tutorial covering what web development is, what web development is comprised of, as well as how it all comes together to create
-web applications. The tutorial mentions installing Atom, don't worry about that. You're going to use VScode instead. 😉
+MacOS: `pwd`
+Windows: `cd`
 
-[HTML & CSS Tutorial (Go Through The Tutorial)](https://internetingishard.com/html-and-css/)
+#### List Directories
+To list all the directories and files in your current location type:
+MacOS : `ls`
+Windows: `dir`
 
-Getting a good grasp of how to layout websites is VERY important. The course below is going to get you moving on just that with some cool new additions to CSS.
+#### Make Directory
 
-<h2 id="javascript">JavaScript</h2>
+We want a place where we can put all our work in. While in the terminal make a new directory called VetsWhoCode by typing mkdir in Windows or MacOS and press enter:
 
-JavaScript is where you go from being an artist to being a wizard in web development. A deep knowledge of JavaScript fundamentals will make you a highly employable developer. Complete Javascript Info to get a good beginner grasp of JS.
+`mkdir VetsWhoCode`
```

Comment:
Where do you want this directory? Students could end up putting it at `/`. I suggest making a `~/code` and `~/code/vetswhocode/` folder. I'd also suggest not using TitleCasing and instead kebab or all lower case. Casing folder names that you need to access via cli gets annoying depending on what computer you're on.